x-pack/plugins/uptime/public/pages/monitor.tsx

```diff
@@ -15,6 +15,20 @@ import { useMonitorId, useUptimeTelemetry, UptimePage } from '../hooks';
 import { MonitorCharts } from '../components/monitor';
 import { MonitorStatusDetails, PingList } from '../components/monitor';
 import { getDynamicSettings } from '../state/actions/dynamic_settings';
+import { Ping } from '../../common/runtime_types/ping';
+
+const isAutogeneratedId = (id: string) => {
+  return /^auto-(icmp|http|tcp)-OX[A-F0-9]{16}-[a-f0-9]{16}/.test(id);
```

Comment:
We should [name this Regex](https://github.com/elastic/kibana/blob/master/STYLEGUIDE.md#name-regular-expressions).